chart/compass/charts/cockpit/OWNERS

```diff
@@ -1,7 +1,10 @@
+reviewers:
+  - console-reviewers
```

Comment:
It won't work as `console-reviewers` are not defined in https://github.com/kyma-incubator/compass/blob/master/OWNERS_ALIASES file. Please copy `console-reviewers` entry from Kyma: 

https://github.com/kyma-project/kyma/blob/master/OWNERS_ALIASES#L18